### PR TITLE
How to sign in to the API via solidus_auth_devise

### PR DESCRIPTION
**Pull request by waiting-for-dev**
## Summary

References https://github.com/solidusio/solidus/issues/4871

I'm for now only providing the version for the edge version of Solidus. I'll copy it to the other versions once we agree on the content.

## Checklist

- [x] I have followed the [Diátaxis](https://diataxis.fr/) framework in my PR.
- [x] I have verified that the preview environment works correctly.

### CODE REVIEW

File: docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    manager.failure_app = AuthFailureApp
+  end
+# ...
+```
+
+After restarting the server, we can now confirm that the response on failure has changed:
+
+```bash
+$ curl -X POST -v -H 'Content-Type: application/json' http://localhost:3000/api/sign_in  -d '{"spree_user": {"email": "admin@example.com", "password": "invalid" }}'
+{"sucess":false,"message":"Invalid email or password."}
```
Comment (kennyadsl) [inline docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx:151]:
```suggestion
{"success":false,"message":"Invalid email or password."}
```

File: docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+```ruby title="lib/auth_failure_app.rb"
+# frozen_string_literal: true
+
+class AuthFailureApp < Devise::FailureApp
+ def http_auth_body
+    return super unless request_format == :json
+
+    {
+      sucess: false,
```
Comment (kennyadsl) [inline docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx:126]:
```suggestion
      success: false,
```

**Review by kennyadsl — COMMENTED** [review]
Found a couple of typos. This is great, thanks, @waiting-for-dev!

File: docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx (hunk 1 above)
Comment (waiting-for-dev) [reply docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx]: Thanks, fixed!

File: docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+  clear_respond_to && respond_to(:json)
+
+  def after_sign_in_path_for(_resource)
+    nil
+  end
+end
+```
+
+There're a few important things to notice here:
```
Comment (elia) [inline docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx:28]: `There're` → `There are`

**Review by elia — APPROVED** [review]
Awesome! 🙌 

(There's still a small typo)

File: docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx (hunk 3 above)
Comment (waiting-for-dev) [reply docs/how-tos/how-to-sign-in-to-the-solidus-api-using-solidus_auth_devise.mdx]: 👍 fixed

**Comment by waiting-for-dev** [comment]
Making links relative to more easily backport to the versioned folder.

**Comment by waiting-for-dev** [comment]
Pushing v3.2 & v3.3

**Pull request merged**